aasrp/static/aasrp/javascript/view-requests.js

```diff
@@ -243,6 +264,11 @@ $(document).ready(() => {
         });
 
         $('.srp-fleet-total-amount').html(`${new Intl.NumberFormat(aaSrpSettings.locale).format(totalSrpAmount)} ISK`);
+
+        //Update tooltip value by first selecting parent and then selecting the icon
+        const ctcElement = element.closest('td').find('i');
+        ctcElement
+            .attr('data-clipboard-text', newValue);
```

Comment:
This is very janky and not bullet proof. `element.closest()` should not be used, no matter what GPT tells you.